### PR TITLE
iteration report archetype & documentation

### PR DESCRIPTION
**Pull request by rlskoeser**
Adds a new post-bundle archetype for iteration reports and instructions in the readme for how to use it.

### CODE REVIEW

**Comment by rlskoeser** [comment]
@thatbudakguy tagged you on this one because I thought it would be good to get another set of eyes on it (and might help you in thinking about archetypes for startwords in future). Not at all urgent, but if you can look it over so we can merge sometime before the end of the iteration it would be nice to use it for next iteration's report.

File: README.md
Comment (thatbudakguy) [inline README.md:39]: Where does the term "post-bundle" come from? I see "page bundle" in the Hugo docs but not sure what this is a reference to - same thing? Would this archetype not just be called "iteration report"?

**Review by thatbudakguy — APPROVED** [review]
seems pretty straightforward! definitely some ways we can use archetypes for startwords.

File: README.md
Comment (rlskoeser) [reply README.md:39]: You're right, it is a page bundle. I wonder where I got post bundle? I guess it's probably clearer to just call it an iteration report archetype? Will revise wording before I merge.

Comment (thatbudakguy) [reply README.md]: It seems like the convention is to call it by the type of post/page it's a template for, so "iteration report archetype" makes sense to me!

**Pull request merged**
